W-15006796 RPA Manager release notes 2024-02/1. (#335)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

[[february-12-2024]]
== February 12, 2024

// == Fixed Issues
include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]

[%header,cols="6a,1a"]
|===
|Issue Resolution |ID

> | The columns _Qualifiers_ and _Benefits_ within the *Process Evaluation* now show correct values. | W-14657848

Reviewer: ```suggestion
| The *Qualifiers* and *Benefits* columns within the *Process Evaluation* module no longer show incorrect values. | W-14657848
```

> | The *Process Streaming* no longer shows a _Gateway Timeout_ error. | W-14826933

Reviewer: ```suggestion
| The *Process Streaming* module no longer shows a _Gateway Timeout_ error. | W-14826933
```

> | A process can now be edited even if the process owner is not specified. | W-14899326

Reviewer: ```suggestion
| You can now edit a process when the process owner is not specified. | W-14899326
```
